src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirOperationDataStore.cs

```diff
@@ -97,7 +74,47 @@ public Task<ExportJobOutcome> GetExportJobByHashAsync(string hash, CancellationT
 
         public Task<ExportJobOutcome> UpdateExportJobAsync(ExportJobRecord jobRecord, WeakETag eTag, CancellationToken cancellationToken)
         {
-            throw new NotImplementedException();
+            // TODO: Implement this method.
+            return Task.FromResult(new ExportJobOutcome(jobRecord, eTag));
+        }
+
+        public async Task<IReadOnlyCollection<ExportJobOutcome>> AcquireExportJobsAsync(ushort maximumNumberOfConcurrentJobsAllowed, TimeSpan jobHeartbeatTimeoutThreshold, CancellationToken cancellationToken)
+        {
+            // We will consider a job to be stale if its timestamp is smaller than or equal to this.
+            DateTimeOffset expirationTime = Clock.UtcNow - jobHeartbeatTimeoutThreshold;
```

Comment:
Is there particular reason why this timestamp is calculated in the FE as opposed to just pass in the threshold and have the BE calculate the cutoff time?

Reply:
Nope, you're right that it makes sense to implement this on the SQL side. John pointed out the same thing. Fixed in d9a5f4b.